iphone/Maps/Maps.xcodeproj/xcshareddata/xcschemes/OMaps.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1610"
+   LastUpgradeVersion = "1620"
```

Comment:
This xcodes autogenerated _updates_ are so annoying! I think that we can leave them

Reply:
They are important in some cases. I'll fix it separately. Some additional module validation for example can be useful.